buildchain/buildchain/ui.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf-8
+
+"""Tasks to build the MetalK8s UI.
+"""
```

Comment:
can be squashed: `"""Tasks to build the MetalK8s UI."""`

or you could a a little schema like:

```
┌────┐    ┌───────┐    ┌───────┐
│ ui │───>│ mkdir │───>│ build │
└────┘    └───────┘    └───────┘
```
as we do for other module.

Reply:
Oh yeah, wanted to add the schema like the other modules, just forgot to do it before pushing. Will do